test/test_core.py

```diff
@@ -5847,6 +5847,14 @@ def test_fs_64bit(self):
       self.set_setting('FORCE_FILESYSTEM')
     self.do_runf('fs/test_64bit.c', 'success')
 
+  @crossplatform
+  @also_with_nodefs_both
```

Comment:
Does this test work with wasmfs?   If do maybe this can be `all_fs`.. is not no worries.

Reply:
Updated.